src/v2/Apps/_Preferences2/PreferencesApp.tsx

```diff
@@ -1,9 +1,182 @@
-import { Text } from "@artsy/palette"
+import {
+  Box,
+  Button,
+  Checkbox,
+  Column,
+  Flex,
+  Separator,
+  Spacer,
+  Text,
+} from "@artsy/palette"
+import { useState } from "react"
 
 export const PreferencesApp: React.FC = () => {
+  const [uncheckedAll, setUncheckedAll] = useState(false)
+
   return (
     <>
-      <Text variant="xl">Preferences App</Text>
+      <Text variant="xl" mt={4}>
+        Preferences Center
+      </Text>
+
+      <Separator mt={2} />
+
+      <Column
```

Comment:
You can't use `Column` without a `GridColumns` wrapper. Take a look at some of the examples: https://palette.artsy.net/elements/gridcolumns/#gatsby-focus-wrapper

Reply:
Updated.